integration_tests/.husky/_/husky.sh

```
@@ -0,0 +1,30 @@
+
+  if [ "$HUSKY" = "0" ]; then
+    debug "HUSKY env variable is set to 0, skipping hook"
+    exit 0
+  fi
+
+  if [ -f ~/.huskyrc ]; then
+    debug "sourcing ~/.huskyrc"
+    . ~/.huskyrc
+  fi
+
+  export readonly husky_skip_init=1
```

Shellcheck (0.10.0)

[error] This applies export to the variable named readonly, which is probably not what you want. Use a separate command or the appropriate `declare` options instead.

(SC2316)